### PR TITLE
testing/apparmor: modernize, fix license

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: testing/apparmor/APKBUILD [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 pkgdesc="Linux application security framework - mandatory access control for programs"
 url="https://gitlab.com/apparmor/apparmor/wikis/home"
 arch="all"
-license="GPL"
+license="GPL-2.0-only"
 depends="bash"
-makedepends="bash sed python3 autoconf automake libtool bison flex perl-dev swig gettext-dev python3-dev linux-pam-dev linux-headers"
+makedepends="sed python3 autoconf automake libtool bison flex perl-dev swig
+	gettext-dev python3-dev linux-pam-dev linux-headers"
 checkdepends="dejagnu py3-pyflakes python2"
```
Comment (TBK) [inline testing/apparmor/APKBUILD]:
```suggestion
checkdepends="dejagnu py3-pyflakes"
```

**Comment by maxice8** [comment]
On Sun, May 26, 2019 at 01:13:11PM -0700, TBK wrote:
> TBK commented on this pull request.
> 
> 
> 
> >  depends="bash"
> -makedepends="bash sed python3 autoconf automake libtool bison flex perl-dev swig gettext-dev python3-dev linux-pam-dev linux-headers"
> +makedepends="sed python3 autoconf automake libtool bison flex perl-dev swig
> +	gettext-dev python3-dev linux-pam-dev linux-headers"
>  checkdepends="dejagnu py3-pyflakes python2"
> 
> ```suggestion
> checkdepends="dejagnu py3-pyflakes"
> ```
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8148#pullrequestreview-242053702

Done

**Comment by Ikke** [comment]
1 test is failing:

> L: test_7 (__main__.AaTest_get_interpreter_and_abstraction)
test '#!/usr/bin/python'
\----------------------------------------------------------------------
Traceback (most recent call last):
File "/home/buildozer/aports/testing/apparmor/src/apparmor-v2.13.2/utils/test/common_test.py", line 88, in stub_test
self._run_test(test_data, expected)
File "test-aa.py", line 187, in _run_test
self.assertEqual(interpreter_path, exp_interpreter_path)
AssertionError: '/usr/bin/python' != '/usr/bin/python2.7'
\- /usr/bin/python
\+ /usr/bin/python2.7
? +++

**Comment by maxice8** [comment]
@Ikke solved

**Pull request closed without merging**